### PR TITLE
Update tracing code, and make it build.

### PR DESCRIPTION
**Pull request by g-easy**

### CODE REVIEW

**Comment by g-easy** [comment]
This only updates the final code, I still need to give the whole doc a once-over.

**Comment by g-easy** [comment]
This is part of #287

File: content/quickstart/cpp/tracing.md [hunk 1]
```diff
@@ -394,62 +395,62 @@ With the above adjustments our sample alongside the updated `BUILD` and `WORKSPA
...
-    return out;
+  span.End();
+  return out;
 }
 
-int main(int argc, char **argv) {
-    // Samplers are potentially expensive to construct. Use one long-lived
-    // sampler instead of constructing one for every Span.
-    static opencensus::trace::AlwaysSampler sampler;
+int main(int argc, char** argv) {
```
Comment (odeke-em) [inline content/quickstart/cpp/tracing.md]: Lol at the `char** argv` vs `char **argv`

Comment (g-easy) [reply content/quickstart/cpp/tracing.md]: I ran it through clang-format. :)

**Pull request merged**